gcp_variant_transforms/testing/integration/small_tests/valid_4_0.json

```diff
@@ -6,14 +6,25 @@
   "assertion_configs": [
     {
       "query": [
-        "SELECT COUNT(0) AS num_rows, ",
-        "  SUM(start_position) AS sum_start, ",
-        "  SUM(end_position) AS sum_end ",
-        "FROM {TABLE_NAME}"
+        "{NUM_ROWS}"
```

Comment:
nit: I think it's cleaner to just put these on one line now that you don't have multiple lines.
e.g.
```
{
  "query": [ "NUM_ROWS_QUERY" ],
  "expected_result": { "num_rows": 5 }
},
{
  "query": [ "SUM_START_QUERY" ],
  "expected_result": { "sum_start": 55 }
},

```

Reply:
Good point!